VorpalEngine/sources/video/Renderer.cpp

```diff
@@ -18,1062 +18,1136 @@ namespace vp
-    if (flags & VK_DEBUG_REPORT_WARNING_BIT_EXT)
-    {
-        LOG_WARNING("VULKAN LAYER WARNING: %s",  msg);
-    };
-    // May indicate sub-optimal usage of the API
-    if (flags & VK_DEBUG_REPORT_PERFORMANCE_WARNING_BIT_EXT)
-    {
-        LOG_INFO("VULKAN LAYER PERFORMANCE: %s",  msg);
-    };
-    // Informal messages that may become handy during debugging
-    if (flags & VK_DEBUG_REPORT_INFORMATION_BIT_EXT)
     {
-        LOG_INFO("VULKAN LAYER INFO: %s",  msg);
-    }
-    // Diagnostic info from the Vulkan loader and layers
-    // Usually not helpful in terms of API usage, but may help to debug layer and loader problems
-    if (flags & VK_DEBUG_REPORT_DEBUG_BIT_EXT)
-    {
-        LOG_INFO("VULKAN LAYER DEBUG: %s",  msg);
+        // Error that may result in undefined behaviour
+        if (flags & VK_DEBUG_REPORT_ERROR_BIT_EXT)
+        {
+            LOG_ERROR("VULKAN LAYER ERROR: %s", msg);
+        };
+        // Warnings may hint at unexpected / non-spec API usage
+        if (flags & VK_DEBUG_REPORT_WARNING_BIT_EXT)
+        {
+            LOG_WARNING("VULKAN LAYER WARNING: %s", msg);
+        };
+        // May indicate sub-optimal usage of the API
+        if (flags & VK_DEBUG_REPORT_PERFORMANCE_WARNING_BIT_EXT)
+        {
+            LOG_INFO("VULKAN LAYER PERFORMANCE: %s", msg);
+        };
+        // Informal messages that may become handy during debugging
+        if (flags & VK_DEBUG_REPORT_INFORMATION_BIT_EXT)
+        {
+            LOG_INFO("VULKAN LAYER INFO: %s", msg);
+        }
+        // Diagnostic info from the Vulkan loader and layers
+        // Usually not helpful in terms of API usage, but may help to debug layer and loader problems
+        if (flags & VK_DEBUG_REPORT_DEBUG_BIT_EXT)
+        {
+            LOG_INFO("VULKAN LAYER DEBUG: %s", msg);
+        }
+        return VK_FALSE;
     }
-    return VK_FALSE;
-}
 
-Renderer::Renderer()
-    : m_isInitialized(false)
-    , m_pWindow(nullptr)
-    , m_vkInstance(VK_NULL_HANDLE)
-    , m_vkPhysicalDevice(VK_NULL_HANDLE)
-    , m_validationLayers( {"VK_LAYER_LUNARG_standard_validation"} )
-    , m_deviceExtensions( {VK_KHR_SWAPCHAIN_EXTENSION_NAME})
-{
-    m_vkSwapChain = VK_NULL_HANDLE;
-    m_renderPass = VK_NULL_HANDLE;
-    m_pipelineLayout = VK_NULL_HANDLE;
-    m_graphicsPipeline = VK_NULL_HANDLE;
-}
-
-Renderer::~Renderer()
-{
-    Deinit();
-}
-
-bool Renderer::Init()
-{
-    if (m_isInitialized)
+    Renderer::Renderer()
+        : m_isInitialized(false)
+        , m_pWindow(nullptr)
+        , m_vkInstance(VK_NULL_HANDLE)
+        , m_vkPhysicalDevice(VK_NULL_HANDLE)
+        , m_validationLayers({ "VK_LAYER_LUNARG_standard_validation" })
+        , m_deviceExtensions({ VK_KHR_SWAPCHAIN_EXTENSION_NAME })
     {
-        return false;
+        m_vkSwapChain      = VK_NULL_HANDLE;
+        m_renderPass       = VK_NULL_HANDLE;
+        m_pipelineLayout   = VK_NULL_HANDLE;
+        m_graphicsPipeline = VK_NULL_HANDLE;
```

Comment:
Not a huge fan of aligning operations. I prefer only comments, but that doesn't bother me much